object-construction-qual/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    // Apply the java plugin to add support for Java
+    id 'java'
+
+    // For bringing in AWS dependencies. We're using the "legacy" version
+    // listed here: https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-project-gradle.html,
+    // because the build fails when we use the Gradle 5+ way, saying that it can't find ec2.
+    // I believe that this is because we need the AWS SDK during testing, but not during the build.
+    id 'io.spring.dependency-management' version '1.0.7.RELEASE'
```

Comment:
Why does the annotation package need the AWS SDK? That doesn't seem right - that should only be necessary for running the `DescribeImagesRequest` tests.